README.rst

```diff
@@ -24,7 +24,7 @@ About
 
 coala-bears is a Python package containing all the bears that are officially
 supported by coala. It features more than **65 bears** covering **35 languages**.
-You can see all of them `here <https://github.com/coala-analyzer/bear-docs>`_.
+`Read more about coala-bears  <https://github.com/coala-analyzer/bear-docs/>`_.
```

Comment:
I'm not so sure about the phrasing here, will people find this if they're searching for supported languages?